packages/@storefront/breadcrumbs/test/unit/breadcrumbs.ts

```diff
@@ -166,4 +175,30 @@ suite('Breadcrumbs', ({ expect, spy, stub, itShouldBeConfigurable, itShouldProvi
       expect(navFields).to.be.eql(['c', 'd']);
     });
   });
+
+  describe('updateSelectedNavigations', () => {
+    it('should set selectedNavigations', () => {
+      // tslint:disable-next-line max-line-length
```

Comment:
I don't think the next line is actually long enough to need this.

Reply:
You're right, removing this. Resolving conversation.